recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -103,6 +103,7 @@ def setup_conda_rc(feedstock_root, recipe_root, config_file):
             update_global_config(feedstock_root)
             channels = _global_config["channels"]["sources"]
 
+        call(["conda", "config", "--remove", "channels", "conda-forge"])
         call(["conda", "config", "--remove", "channels", "defaults"])
         for c in reversed(channels):
             call(["conda", "config", "--add", "channels", c])
```

Comment:
Simplification:
```suggestion
        call(["conda", "config", "--remove-key", "channels"])
        for c in channels:
            call(["conda", "config", "--append", "channels", c])
```

Reply:
Ah, wait I think it'll auto-readd `defaults` if the key is empty.
This should work universally:
```suggestion
        call(["conda", "config", "--remove-key", "channels"])
        for c in channels:
            call(["conda", "config", "--append", "channels", c])
        if "defaults" not in channels:
            call(["conda", "config", "--remove", "channels", "defaults"])
```

Reply:
Thanks